Formula/dronedb.rb

```diff
@@ -24,6 +24,12 @@ class Dronedb < Formula
   depends_on "libzip"
   depends_on "pdal"
 
+  # To fix gdal-3.7.0
+  patch do
+    url "https://github.com/e-n-f/DroneDB/commit/28aa869dee5920c2d948e1b623f2f9d518bdcb1e.patch?full_index=1"
+    sha256 "50e581aad0fd3226fe5999cc91f9a61fdcbc42c5ba2394d9def89b70183f9c96"
+  end
```

Comment:
Has this been submitted upstream?

Reply:
the patch did get merged per this PR, https://github.com/DroneDB/DroneDB/pull/374